src/ray/raylet_client/raylet_client.cc

```diff
@@ -296,6 +297,37 @@ Status raylet::RayletClient::SetResource(const std::string &resource_name,
   return conn_->WriteMessage(MessageType::SetResourceRequest, &fbb);
 }
 
+/// Spill objects to external storage.
+/// \param object_ids The IDs of objects to be spilled.
+/// \return Status
+Status raylet::RayletClient::SpillObjects(const std::vector<ObjectID> &object_ids) {
+  flatbuffers::FlatBufferBuilder fbb;
+  auto message = protocol::CreateSpillObjectsRequest(fbb, to_flatbuf(fbb, object_ids));
+  fbb.Finish(message);
+  std::vector<uint8_t> reply;
+  RAY_RETURN_NOT_OK(conn_->AtomicRequestReply(
+      MessageType::SpillObjectsRequest, MessageType::SpillObjectsReply, &reply, &fbb));
+  RAY_UNUSED(flatbuffers::GetRoot<protocol::SpillObjectsReply>(reply.data()));
```

Comment:
Instead of adding a legacy Raylet client RPC can we make these gRPC calls instead?

Reply:
It seems all other blocking calls are implemented with non-gRPC calls.

Reply:
Please change this to a gRPC call, we will be removing the legacy TCP calls in the future. If you really want it to be blocking, you can add that on the async call by creating an `absl::Notification` and completing that when the callback finishes.